src/Contributors/Model/Contributor.php

```diff
@@ -39,7 +39,8 @@ public function getProfileUrl(): string
 
     public function getProfileImg(): string
     {
-        return $this->profileImg.'?s='.$this->size;
+        $sep = strpos($this->profileImg, '?') ? '&' : '?';
```

Comment:
Is it possible to simplify this line to have a more readable code? Maybe split into more lines or create a separate function. What do you think?